README.md

```diff
@@ -22,7 +22,7 @@
 
 ## 📖 Personal Portfolio <a name="about-project"></a>
 
-A mobile view portfolio showing only the header and headline section. The headline section shows the summary of the author's information
+A mobile view portfolio showing only the header and headline section. The headline section shows the summary of the author's information. The work section display list of projects I have worked on and about section shows information about myself
 
 ### 🛠 Built With <a name="built-with"></a>
 
```

Comment:
- [ ] I was about to approve your project because it's perfectly implemented :clap: , until I notice that you did not have all the main headers from the [README template](https://github.com/microverseinc/readme-template)headers from the [README template](https://github.com/microverseinc/readme-template). Kindly update your README file to pass to the next step :next_track_button:

Reply:
Hello @iambenkis 
I have already customised the [README Template](https://github.com/microverseinc/readme-template) to suit what I have in my project just as we were instructed to during our first project. I already have the headers that apply to what I have done currently and I update it as I add more features to it.